### PR TITLE
Issue/65/conversions

### PR DESCRIPTION
**Pull request by aimalz**
I'm starting a pull request because I implemented the one conversion we need for the data exploration notebook, from the gridded approximation to samples (via rejection sampling).  However, I'd like to continue working on this branch at some point in the future (for the "Someday. . ." milestone?) for completeness, as the following conversions are still unsupported:

- gridded --> quantiles
- gridded --> histogram
- samples --> quantiles
- samples --> histogram
- quantiles --> histogram
- quantiles --> functional form
- histogram --> quantiles
- histogram --> functional form

### CODE REVIEW

File: docs/notebooks/demo.ipynb [hunk 1]
```diff
@@ -230,6 +230,7 @@
     "gridded = P.evaluate(grid, using='truth', vb=False)\n",
     "\n",
     "G = qp.PDF(gridded=gridded)\n",
+    "G.sample(100, vb=False)\n",
```
Comment (drphilmarshall) [inline docs/notebooks/demo.ipynb:233]: Sweet.

**Review by drphilmarshall — APPROVED** [review]
Looks good to me, although I didn't run the notebook. Let's press on - back to the DESC Note to try approximating a subset of the PDFs! :-)

**Comment by drphilmarshall** [comment]
You can merge and leave the branch undeleted if you like, although TBH I'd not be worried about forgetting about your wishlist and just making improvments as you need them. Right now we have data to analyze!

**Comment by aimalz** [comment]
Because I didn't implement all the conversions in #67 , things are getting very messy in #43.  I should either resurrect #65 and finish the job or write in a lot of checks guaranteeing that all members of the ensemble being parametrized from format X do so starting from the same format Y, for uniformity.

**Pull request merged**